changelog/fix_style_symbol_array_false_positives.md

```diff
@@ -0,0 +1 @@
+* [#12071](https://github.com/rubocop/rubocop/issues/12071): Fix `Style/SymbolArray` false positives. ([@jasondoc3][])
```

Comment:
Can you add a little more information to the description about what will be fixed?

Reply:
Added a bit more detail here